cedar-policy/src/api.rs

```diff
@@ -1852,6 +1852,12 @@ impl SourceLocation {
     pub fn range_end(&self) -> Option<usize> {
         self.source_loc.as_ref().map(parser::Loc::end)
     }
+
+    /// Returns a tuple of (offset, length) of the location.
+    /// Returns `None` if this location does not have a range.
+    pub fn offset_and_length(&self) -> Option<(usize, usize)> {
+        self.source_loc.as_ref().map(|loc| (loc.start(), loc.end()))
+    }
```

Comment:
The doc comment and method name say (offset, length), but the code does (start, end).

Reply:
Great pickup, thanks I missed this in the initial PR. I've fixed this up.

Reply:
I see that Miette uses offset and length, whereas `cedar_policy_core::parser::Loc` uses start and end. I've changed the `SourceLocation` struct I added to use start and end in order to align with `cedar_policy_core::parser::Loc`.